### PR TITLE
Create a devcontainer image

### PR DESCRIPTION
**Pull request by jbtrystram**
Provide a ready to go development environment :

```shell
[seedwing-policy] toolbox create seedwing --image ghcr.io/jbtrystram/devcontainer
Created container: seedwing
Enter with: toolbox enter seedwing
[seedwing-policy] toolbox enter seedwing   
⬢[jibou@toolbox seedwing-policy]$
⬢[jibou@toolbox seedwing-policy]$ cd seedwing-policy-server/
⬢[jibou@toolbox seedwing-policy-server]$ cargo run
   Compiling seedwing-policy-server v0.1.0-alpha.2 (/home/jibou/github/seedwing/seedwing-policy/seedwing-policy-server)
    Finished dev [unoptimized + debuginfo] target(s) in 7.11s
```

### CODE REVIEW

**Comment by jcrossley3** [comment]
TIL about `toolbox`. :smile: Very cool!

**Comment by jbtrystram** [comment]
@jcrossley3 toolbox is great ! And absolutely indispensable if you use an immutable OS (eg fedora silverblue)

File: .github/workflows/dev_container.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+name: Developpement container
```
Comment (lulf) [inline .github/workflows/dev_container.yaml:1]: Development

**Review by bobmcwhirter — APPROVED** [review]

File: .github/workflows/dev_container.yaml (hunk 1 above)
Comment (jbtrystram) [reply .github/workflows/dev_container.yaml:1]:
```suggestion
name: Development container
```

**Pull request merged**